skeletor/export/css.config.js

```diff
@@ -10,8 +33,13 @@ module.exports = {
                 files: [
                     ...files(`${outputDir}`)
                 ],
-                plugins
+                plugins: [
+					...plugins,
+					require('postcss-url')({
+						url: mapUrl
+					}),
```

Comment:
[question] I can't remember, will node will throw an error if you end an array with a trailing comma?

Reply:
Not 100% sure, so it is removed